activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -606,7 +606,8 @@ def order!(*args) # :nodoc:
       self
     end
 
-    # Allows to specify an order by a specific set of values.
+    # Allows to specify an <code>ORDER BY</code> clause to a query based on a given +column+,
+    # ordered and filtered by the specific set of +values+.
```

Comment:
I think the first sentence can be simplified a bit.
I'm not a fan of the `Allows to ..` as used by other methods in this file.
Removing it makes things simpler and clearer.

We also prefer `<tt>` for fixed width fonts:
https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#fixed-width-font

I also don't think `in_order_of` filters the results.
```suggestion
    # Specify an <tt>ORDER BY</tt> clause to a query based on a given +column+,
    # ordered by a specific set of +values+.
```

I find "Specifying a clause to a query" a bit wordy still.
Maybe we can remove the "to a query" part as well.
```suggestion
    # Specify an <tt>ORDER BY</tt> clause based on a given +column+,
    # ordered by a specific set of +values+.
```

Or maybe making it similar to `order`:
```suggestion
    # Applies an <tt>ORDER BY</tt> clause to a query based on a given +column+,
    # ordered by a specific set of +values+.
```

Reply:
My bad, I didn't know it filters as well.

Reply:
yeah, the filtering behavior was a bit surprising to me as well. 😄

Reply:
updated!